### PR TITLE
Feature/vudrm 1343/release notes

### PR DESCRIPTION
**Pull request by PerritonBranch**
Probably best to to view the docs on this to check how the release notes.
https://docs.vualto.com/projects/vudrm/en/feature-vudrm-1343-release-notes/Release-notes.html

### CODE REVIEW

**Comment by somerobmitchell** [comment]
Needs to wait for actual release notes before merging.

**Review by rogerpales — DISMISSED** [review]

**Review by fschurer — DISMISSED** [review]

**Review by VUjackcornelius — DISMISSED** [review]

**Review by somerobmitchell — APPROVED** [review]

**Review by fschurer — APPROVED** [review]

**Pull request merged**
